src/App.js

```diff
@@ -5,17 +5,25 @@
 */
 
 // Import the state hook
-import React from 'react';
+import React, { useState } from "react";
 // Import the Posts (plural!) and SearchBar components, since they are used inside App component
+import Posts from "./components/Posts/Posts";
+
+import SearchBar from "./components/SearchBar/SearchBar";
 // Import the dummyData
-import './App.css';
+
+import "./App.css";
+import DummyData from "./dummy-data";
```

Comment:
Try to only capitalize components (ex: line 16 dummyData instead of DummyData)so you won't get confused. Still works, but it is better syntax/practice